BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Models/SkontoViewModel.swift

```diff
@@ -124,11 +126,16 @@ class SkontoViewModel {
         return text
     }
 
+    private var errorMessage: String?
+
     weak var delegate: SkontoViewModelDelegate?
 
-    init(skontoDiscounts: SkontoDiscounts, isWithDiscountSwitchAvailable: Bool = true) {
+    init(skontoDiscounts: SkontoDiscounts,
+         isWithDiscountSwitchAvailable: Bool = true,
+         maximumAmountToPayValue: Decimal = 99999.99) {
```

Comment:
[suggestion] Can we have the maximumAmountToPayValue of 99999.99 as a let property?

Reply:
in the PR with PP-906 I have implemented `func setMaximumAmountToPayValue(_ value: Decimal?)` which changes this value for Skonto + RA, thats the reason of `var` here